src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/UpdateResourcesValidation.feature

```diff
@@ -4,7 +4,7 @@ Feature: Resources "Update" Operation validations
         Background:
             Given the Data Management Service must receive a token issued by "http://localhost"
               And user is already authorized
-              And request made to "data/ed-fi/absenceEventCategoryDescriptors" with
+              And a POST request is made to "data/ed-fi/absenceEventCategoryDescriptors" with
```

Comment:
Prefer that the URL not include "data" since that is not part of the Ed-Fi API specification. Will have to add "data" in the step definition C#.

Reply:
Addressed.